packages/form/src/cli.ts

```diff
@@ -26,17 +26,70 @@ switch (cmd) {
 async function create() {
 	intro(`Create a new form component`);
 	// Do stuff
-	const useReact = handleCancel(
-		await confirm({
-			message: "Do you want to use React?",
-			initialValue: true,
-		}),
-	);
-
-	if (!useReact) {
-		note("Well, sorry. We only support React right now ;)");
+	const frameworks = [
+		// ? this code should be here?
+		{
+			value: "react",
+			label: "React",
+			templatePath: "react/Form.tsx",
+			componentName: "Form.tsx",
+		},
+		{
+			value: "preact",
+			label: "Preact",
+			templatePath: "preact/Form.tsx",
+			componentName: "Form.tsx",
+		},
+	] as const;
```

Comment:
I think this is a good start! I'd hoist to the file root just for clarity. Also, We _might_ want to model this for a future where a framework copies multiple files. For example, the Vue version will probably copy a `Form.vue` and an `Input.vue`. I wonder if we could model like this:

```ts
const frameworks = [
		{
			value: "react",
			label: "React",
			templateDir: "react",
		},
```

And update the copy prompts to:
1. Read all files in that `templateDir`, excluding internal files (`env.d.ts` and `tsconfig.json`)
2. Loop over the files and check if any collide with files in the user's output dir. If there is, present a clear error message. Maybe "Cannot copy files without name conflicts. Choose a different directory"
3. Once the user gives a directory without conflicts, copy the template dir contents to their output 1-1. May want to [use the `fs-extra` package](https://www.npmjs.com/package/fs-extra) to make copying easier.